docs/source-2.0/aws/aws-cloudformation.rst

```diff
@@ -659,63 +659,72 @@ The following example defines a CloudFormation resource that has the
 --------------------------------------------
 
 Summary
-    Indicates that the member annotated has a default value for the resource.
+    Indicates that the member annotated has a default value
+    for that property of the CloudFormation resource. Thus,
+    when this trait annotates an ``@output`` structure member,
+    it indicates that the CloudFormation property generated
+    from that member has a default value in the CloudFormation
+    schema. This trait can be used to indicate that an output
+    field with a value may return a default value assigned
+    by the service.
 Trait selector
     ``resource > operation -[output]-> structure > member``
 
-    *Only applicable to members of ``@output`` operations*
+    *Only applicable to members of @output operations*
 Value type
     Annotation trait
 
-Given the following example, because the ``fooAlias``
+Given the following example, because the ``backTrackWindow``
 member is annotated with ``cfnDefaultValue``, it can be derived
-that the ``fooAlias`` member has a default value for this resource.
+that the ``backTrackWindow`` member has a default value in the
+CloudFormation schema for this resource.
 
 .. code-block:: smithy
 
     $version: "2"
 
     namespace smithy.example
 
-    use aws.cloudformation#cfnDefaultValue
+   use aws.cloudformation#cfnDefaultValue
     use aws.cloudformation#cfnResource
 
+    // this is a very simplified example of AWS::RDS::DBCluster
     @cfnResource
-    resource Foo {
+    resource DBCluster {
         identifiers: {
-            fooId: String
-        }
-        read: GetFoo
+            DBClusterIdentifier: String,
+        },
+        create: CreateDBCluster,
+        read: DescribeDBCluster,
+        update: ModifyDBCluster,
+        delete: DeleteDBCluster
     }
 
     @readonly
-    @http(method: "GET", uri: "/foos/{fooId}", code: 200)
-    operation GetFoo {
-        input: GetFooRequest
-        output: GetFooResponse
+    @http(method: "GET", uri: "/rds/{DBClusterIdentifier}", code: 200)
+    operation DescribeDBCluster {
+        input: DescribeDBClusterRequest,
+        output: DescribeDBClusterResponse,
     }
 
     @input
-    structure GetFooRequest {
+    structure DescribeDBClusterRequest {
         @httpLabel
         @required
-        fooId: String
-
-        fooAlias: String
+        DBClusterIdentifier: String,
```

Comment:
I think you can get away with removing most of this defining only the relevant resource and structure shape information (also using the IDL2 upcoming resource-properties and elision syntax):

```
resource DBCluster {
    identifiers: { DBClusterIdentifier: String }
    properties: { backTrackWindow: Integer }
}

structure DescribeDBClusterResponse for DBCluster {
    $DBClusterIdentifier
    
        // If the user has not specified backTrackWindow
        // upon create, a default value will still be observed
        // in the DescribeDBClusterResponse
    @cfnDefaultValue
    $backTrackWindow
```

Minor adjustments to the wording

Reply:
Makes sense! I was unsure if it would be helpful to have the operation and input structures defined to give a more clear picture of this. 

Cleaned up the example and syntax and working in Commit 4.